src/java/org/apache/cassandra/net/MessagingService.java

```diff
@@ -215,7 +217,7 @@ public class MessagingService extends MessagingServiceMBeanImpl
     // DSE 6.8 version for backward compatibility
     public static final int VERSION_DSE_68 = 168;
```

Comment:
@jtgrabowski do you know where is `168` coming from?

Reply:
Still trying to figure this out. It was added here: https://github.com/datastax/cassandra/pull/316/files#diff-7284e8da2bb0ea3e0e965a8f728407b2326f982281d6b37ca0d7cad79039b70eR213. I'll follow up with Kuba.

Reply:
In fact let me ask @jakubzytka here. Kuba where does 168 come from? Zhao and I can't seam to find where it's defined in DSE.

Reply:
TLDR: I think that batchlog version detection doesn't work correctly and was left in a placeholder state so that we can fix it quickly and have a working upgrade once it's needed.

Longer story:
After going back to STAR-985 I think this `168` is just a placeholder.
In STAR-985 we connected batch writing hook in the DB to CNDB's batchlog writing machinery.
Along the way came the serialization change from DSE (`maxTimestamp`). CC-based CNDB is supposed to be able to deserialize DSE-based batchlog data.
However, IIUC the batchlog version detection code was left unchanged, so it doesn't work in practice (i.e. fix version detection and upgrade should work)

Does ^ make sense?

Reply:
> I think that batchlog version detection doesn't work correctly and was left in a placeholder state so that we can fix it quickly and have a working upgrade once it's needed.

IMO what's important here is that `BatchlogData` serde works. It is based on `BatchlogData$SerializationVersion` which is defined in the same way in CC and DSE (enum oridnal). The `168` is effectively only used in CC and denotes that `maxTimestamp` should be skipped. IMO we could change the value and batch logs wouldn't break.